changelog/deprecateNonPropertyAssignment.dd

```diff
@@ -0,0 +1,16 @@
+the property assignment to non-@property functions has been deprecated.
```

Comment:
such special casing will lead us a c++-like forest of rules and well special cases